test/services/MeasureService.test.ts

```diff
@@ -137,7 +142,7 @@ describe('MeasureService', () => {
         });
     });
 
-    it('returns a Bundle including the root lib, Measure, and  when root lib has dependencies and id passed through body', async () => {
+    it('returns a Bundle including the root lib, Measure, and when root lib has no dependencies and id passed through body', async () => {
```

Comment:
```suggestion
    it('returns a Bundle including the root lib and Measure when root lib has no dependencies and id passed through body', async () => {
```
I think this is what we want for the test description? Unless something else is supposed to come after the "and" that I'm missing